lib/web_ui/lib/src/engine/util.dart

```diff
@@ -339,20 +339,15 @@ bool rectContainsOther(ui.Rect rect, ui.Rect other) {
       rect.bottom >= other.bottom;
 }
 
-/// Converts color to a css compatible attribute value.
-String? colorToCssString(ui.Color? color) {
-  if (color == null) {
-    return null;
+extension CssColor on ui.Color {
+  /// Converts color to a css compatible attribute value.
+  String toCssString() {
+    return colorValueToCssString(value);
   }
-  final int value = color.value;
-  return colorValueToCssString(value);
 }
 
 // Converts a color value (as an int) into a CSS-compatible value.
-String? colorValueToCssString(int? value) {
-  if (value == null) {
-    return null;
-  }
+String colorValueToCssString(int value) {
```

Comment:
Should this be made private? Or do we still have usages of this method where value is a `0xFFFABADA`?

Reply:
This is still used in other files when we don't have a `ui.Color` instance but we have an `int` representation of the color. Examples:
https://github.com/flutter/engine/blob/4d507067285936553780967e3fda4dabd8258fc3/lib/web_ui/lib/src/engine/html/bitmap_canvas.dart#L575
https://github.com/flutter/engine/blob/9322c14c1d59a3c957f34f3f25c9f15b3c5a3016/lib/web_ui/lib/src/engine/canvas_pool.dart#L1000